modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -32,7 +33,22 @@ import {
 import {BAR_CHART, COLORS, DEFAULT_COLOR} from '../utils/constants';
 import {shortenNumber} from '../utils/shortenNumber';
 
-export default function AuditBarChart({rtl, vocabularies}) {
+const ESCAPE_KEYS = [
+	'Escape', // Most browsers.
+	'Esc', // Edge.
+];
```

Comment:
(Gah... GitHub seems to be eating my comments... one more try... sorry if this ends up being a dupe.)

---

Don't use an array when your use case is checking for membership in a collection.

The `O(n)` vs `O(1)` thing obvious doesn't matter here, but it's important to get in the habit of using the right data structure for the job at hand, so that when you find yourself in a situation where `n` is large, you're already doing the right thing.

```js
const ESCAPE_KEYS = new Set(['Esc', 'Escape']);

// and later...
if (ESCAPE_KEYS.has(event.key) ...)
```

(may as well sort Esc before Escape while you're at it).

Reply:
Thanks for the feedback Greg! You're right, I have to me more friendly with `Set` 😛 . I'll aply and push the changes. Thanks!